docs/install_guides/installing-model-dev-tools.md

```diff
@@ -96,7 +96,7 @@ You will see the help message for Model Optimizer if installation finished succe
 
 Note the following things:
 
-* To install OpenVINO Development Tools, you must have OpenVINO Runtime installed first. You can install OpenVINO Runtime through an installer ([Linux](installing-openvino-linux.md), [Windows](installing-openvino-windows.md), or [macOS](installing-openvino-macos.md)), [APT for Linux](installing-openvino-apt.md) or [YUM for Linux](installing-openvino-yum.md). 
+* To install OpenVINO Development Tools, you must have OpenVINO Runtime installed first. You can install OpenVINO Runtime through archive files. See [Linux installation](installing-openvino-linux-header.md), [Windows installation](installing-openvino-windows-header.md), and [macOS installation](installing-openvino-windows-header.md) for more details. 
```

Comment:
```suggestion
* To install OpenVINO Development Tools, you must have OpenVINO Runtime installed first. You can install OpenVINO Runtime through archive files. See [Install OpenVINO on Linux](installing-openvino-linux-header.md), [Install OpenVINO on Windows](installing-openvino-windows-header.md), and [Install OpenVINO on macOS](installing-openvino-windows-header.md) for more details. 
```
Sorry I realized this could be misleading.